src/main/resources/conf/shifuconfig

```diff
@@ -40,7 +40,7 @@ guagua.zk.embedbed.isInClient=true
 
 # if over such threshold three times, kill worker; by default is 60000, change it to 30 minutes to avoid failures
 # set worker compute time threshold to 30 minutes, if compute time of 3 iterations >= this value, worker will kill itself.
-guagua.computation.time.threshold=1800000
+# guagua.computation.time.threshold=1800000
```

Comment:
Thanks Mark for such PR, could you share why such parameter commented?

Reply:
The configuration is duplicated here https://github.com/ShifuML/shifu/blob/develop/src/main/resources/conf/shifuconfig#L114